src/Web/Sfa.Das.Sas.Infrastructure/ElasticSearch/ElasticsearchClientFactory.cs

```diff
@@ -16,16 +19,31 @@ public ElasticsearchClientFactory(IConfigurationSettings applicationSettings)
 
         public IElasticClient Create()
         {
-            using (var settings = new ConnectionSettings(new StaticConnectionPool(_applicationSettings.ElasticServerUrls)))
+            ConnectionSettings settings;
```

Comment:
I can't see where this 'ConnectionSettings' IDisposable type has .Dispose called now it's not longer in a using

Can you check this isn't a leak?